store/iavl/store.go

```diff
@@ -281,7 +281,7 @@ func (st *Store) Export(version int64) (*iavl.Exporter, error) {
 	if !ok || tree == nil {
 		return nil, fmt.Errorf("iavl export failed: unable to fetch tree for version %v", version)
 	}
-	return tree.Export(), nil
+	return tree.Export()
```

Comment:
API break that was from v0.19.4 -> v0.19.5. 
tree.Export now returns both the export and error thus no need to handle error separately in return statement